mobile/src/main/java/ch/epfl/sdp/mobile/state/StatefulGameScreen.kt

```diff
@@ -68,8 +71,33 @@ fun StatefulGameScreen(
 }
 
 /**
- * An implementation of [GameScreenState] that starts with default chess positions, can move pieces
- * and has a static move list.
+ * A composable which displays a full-screen dialog for the [PromotionState].
+ *
+ * @param state the [PromotionState] which backs this dialog.
+ * @param modifier the [Modifier] for this composable.
+ */
+@Composable
+private fun StatefulPromoteDialog(
+    state: PromotionState,
+    modifier: Modifier = Modifier,
+) {
+  if (state.choices.isNotEmpty()) {
+    PromoteDialog(
+        // Always use the Black color to remain consistent with Figma ?
+        color = ChessBoardState.Color.Black,
```

Comment:
Although it's clearly not a priority, I think it would be good to have a low-priority issue for displaying the correct color here.

Reply:
Referenced in https://github.com/epfl-SDP/android/issues/265.